packages/net/http/payload.pony

```diff
@@ -243,3 +242,26 @@ class iso Payload
     end
 
     list
+
+  fun string(): String =>
+    var str = proto + status.string() + method + "\n"
+    for (k, v) in headers().pairs() do
+      str = str + k + ": " + v + "\n"
+    end
+    str = str + "\n"
+    for chunk in body().values() do
+      str = str + match chunk
+      | let s: String val => s
+      | let bs: Array[U8 val] val => String.from_array(bs)
+      else
+        ""
+      end
+    end
+    str = str + "\n"
+
+primitive Content
```

Comment:
The primitive can be alternatively called `MediaType` to match naming in [RFC7231][1] (emphasis added):
> The "Content-Type" header field indicates the **media type** of the associated representation <...>

[1]: http://tools.ietf.org/html/rfc7231#section-3.1.1.5